### DISCUSSION TITLE
"An invalid response was received from the upstream server"

### DISCUSSION THREAD

**Discussion by prakash-yemineni** [Kong/kong#9939] [Help]
I created a service(camunda's login page) which is running on my localhost:8080 and I created routing with path  http://localhost:8000/camunda but I am getting the error saying that "An invalid response was received from the upstream Server" Please help me.

**Comment by PalAditya** [comment]
Can you share your configuration? Also, are you certain that your endpoint is returning a valid response with proper headers when accessed directly (without Kong)?
